### ISSUE TITLE
Disabling?

### ISSUE THREAD

**Issue by reintroducing** [alsoscotland/react-super-select#75]
First off let me say this component is fantastic, nice job.

One thing so far that I've found it is missing is the ability to disable it much like you can disable a regular Select dropdown.  It would be nice to just pass a `disabled` prop to ReactSuperSelect.  That would make it easy and the API intuitive (the same as a regular form element).

Thoughts?

**Comment by alsoscotland** [comment]
@reintroducing 
Absolutely.  That is an oversight on my part.  I will look into adding that this week.

**Comment by alsoscotland** [comment]
https://github.com/alsoscotland/react-super-select/pull/78
